src/main/java/org/springframework/data/redis/core/RedisTemplate.java

```diff
@@ -79,12 +79,15 @@
  * @author Ninad Divadkar
  * @author Anqing Shao
  * @author Mark Paluch
+ * @author Christian Bühler
  * @param <K> the Redis key type against which the template works (usually a String)
  * @param <V> the Redis value type against which the template works
  * @see StringRedisTemplate
  */
 public class RedisTemplate<K, V> extends RedisAccessor implements RedisOperations<K, V>, BeanClassLoaderAware {
 
+	public static final int REDIS_MAX_ARGS = 1024 * 1024;
```

Comment:
We should remove this one as there's no need for the constant (see comment above about atomicity).

Reply:
Yes, removed.